src/maskAnalysis/getFeret.ts

```diff
@@ -73,7 +96,16 @@ export function getFeret(mask: Mask): Feret {
   let minWidth = Number.POSITIVE_INFINITY;
   let minWidthAngle = 0;
   let minLinePoints: Point[] = [];
-
+  let minLines: [[Point, Point], [Point, Point]] = [
```

Comment:
You should not define minLines if you are regenerating a new object later in your function.